addons/website/views/snippets/snippets.xml

```diff
@@ -309,7 +309,32 @@
     </we-multi>
 </template>
 
+<template id="option_header_brand">
+    <we-select string="Type" data-reload="/">
+        <we-button data-customize-website-views="" data-name="option_header_brand_none">None</we-button>
+        <we-button data-customize-website-views="website.option_header_brand_name">Text</we-button>
+        <we-button data-customize-website-views="website.option_header_brand_logo">Image</we-button>
+    </we-select>
+</template>
+
 <template id="snippet_options">
+    <div data-selector="#wrapwrap > header nav.navbar .o_logo" data-no-check="true">
+        <t t-call="website.option_header_brand"/>
+        <we-input string="Height"
+                  data-dependencies="!option_header_brand_none"
+                  data-customize-website-variable="null"
+                  data-variable="logo-height"
+                  data-unit="px"
+                  data-save-unit="rem"/>
+        <we-input string="Height (Scrolled)"
+                  data-name="option_logo_height_scrolled"
+                  data-customize-website-variable="null"
+                  data-variable="fixed-logo-height"
+                  data-unit="px"
+                  data-save-unit="rem"
+                  data-dependencies="!header_effect_scroll_opt"/>
+    </div>
```

Comment:
Why added here before web_editor options? Why not adding it at a meaningful place next to other header options?

Reply:
I made a call with SBU he wanted it at the end.

Reply:
Your reply has nothing to do with what I said...

1. I am commentating an option you put at the very top, not at the end
2. I am commentating on the position *in the code*, nothing to do with SBU.